docs/guide/topics/permissions.rst

```diff
@@ -0,0 +1,219 @@
+    overwrites[guild.default_role] = discord.PermissionOverwrite(view_channel=False, send_messages=False, read_messages=False, read_message_history=False)
+    overwrites[moderator_role] = discord.PermissionOverwrite(manage_messages=True)
+    overwrites[member] = discord.PermissionOverwrite(view_channel=True, send_messages=True, read_messages=True, read_message_history=True)
+
+    # We can now pass these to channel creation:
+
+    await guild.create_text_channel(name="Ticket", overwrites=overwrites)
+
+The channel created whose permissions look something like this:
+
+.. image:: /images/guide/topics/permissions/ticket_channel.png
+
+Using ``PermissionOverwrite`` objects for finer control over channels
+----------------------------------------------------------------------
+
+:class:`~PermissionOverwrite` makes granting finer access to channels easier to manipulate.
+Let's create a scenario where you want to "lockdown" a channel, denying the ``@everyone`` role the permission to send messages.
+
+.. code-block:: python3
+
+    # I have found that the easiest way to do this, is to get the current overwrites for the Role or Member we're editing:
+    overwrite = channel.overwrites_for(guild.default_role)
+    # We now have the PermissionOverwrite instance for the `@everyone` role, in the channel.
+
+    # All we do now, is update it. There are two ways to do this:
+    overwrite.send_messages = False
+    # or, if you have bulk permissions to update:
+    overwrite.update(send_messages=False)
+
+    # Since we have a singular PermissionOverwrite, we want to use the `set_permissions` method of the channel:
+    await channel.set_permissions(guild.default_role, overwrite=overwrite)
+
+Now, that channel has explicitly denied the ``send_messages`` permission to the defalt role of the guild.
+
+In an alternate scenario, we want to add multiple overwrites to a channel at the same time, to deny ``add_reactions`` permissions to roles and members:
+
+.. code-block:: python3
+
+    # Let's take a working copy of the overwrites in that channel
+    overwrites = channel.overwrites.copy()
+
+    # we'll now grab the specific overwrites relating to that role and member:
+    role_overwrite = channel.overwrites_for(role)
+    member_overwrite = channel.overwrites_for(member)
+
+    # edit the attribute for each specific permission
+    role_overwrite.add_reactions = False
+    member_overwrite.add_reactions = False
+    # alternatively, if you have a bulk of permissions to update, you can use the `.update` method of them:
+    role_overwrite.update(add_reaction=False)
+    member_overwrite.update(add_reaction=False)
+
+    # We now place these overwrites into the existing and working copy of the overwrites we took:
+    overwrites[role] = role_overwrite
+    overwrites[member] = member_overwrite
+
+    # and edit the channel with them:
+    await channel.edit(overwrites=overwrites)
```

Comment:
I discussed this on other guide reviews, but I think it is always best practice to end a guide with prose as opposed to a codeblock/warning/other directive, because it makes the document feel better rounded off.

Reply:
I do agree, but I am not sure what to add here personally. Similar to the other remaining conversation I will await feedback from other contributors.